README.md

```diff
@@ -1,2 +1,160 @@
-# simplify-redux
- Redux simplify
+# 🐒  Simplify redux ap
+<p align="center">
+  <a href="https://github.com/simplify-apps/simplify-redux-app/actions/workflows/publish.yml">
+    <img src="https://github.com/simplify-apps/simplify-redux-app/actions/workflows/publish.yml/badge.svg" />
+  </a>
+
+  <a href="https://npm.im/simplify-redux-app/">
+    <img src="https://img.shields.io/npm/v/simplify-redux-app.svg" />
+  </a>
+
+  <a href="https://npm.im/reaviz">
+    <img src="https://badgen.net/npm/dw/simplify-redux-app" />
+  </a>
+
+  <a href="https://github.com/simplify-apps/simplify-redux-app/blob/master/LICENSE">
+    <img src="https://badgen.now.sh/badge/license/MIT" />
+  </a>
+  
+  <a href="https://bundlephobia.com/result?p=simplify-redux-app">
+    <img src="https://badgen.net/bundlephobia/minzip/simplify-redux-app">
+  </a>
+</p>
+
+Simplify redux app (SRA) is the modern and lightweight (less than 2 kb) toolkit that will simplify your work with redux. SRA allows you to forget all pain with actions, constants, reducers, moreover, it is fully compatible with existing code.
+
+The library is actively used and developed, so if you have any questions welcome! 👋 
+
+
+## 📦 Installation
+
+To use SRA in your project, install it via npm:
+
+```
+npm i simplify-redux-app
+```
+or by yarn:
+```
+yarn add simplify-redux-app
+```
+
+then create module
+
+```jsx
+import { simplifyBuilder } from 'simplify-redux-app';
+
+// create builder that will handle all works
```

Comment:
// create builder that will handle all the work

Reply:
thx